src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.MultiSegment.cs

```diff
@@ -2355,25 +2354,48 @@ private bool ConsumeCommentMultiSegment()
             }
 
             byte marker = localBuffer[0];
-
-            if (marker == JsonConstants.Slash)
+            if (marker != JsonConstants.Slash && marker != JsonConstants.Asterisk)
             {
-                return ConsumeSingleLineCommentMultiSegment(localBuffer.Slice(1), leftOver, start, _consumed);
+                ThrowHelper.ThrowJsonReaderException(ref this, ExceptionResource.InvalidCharacterAtStartOfComment, marker);
```

Comment:
Same here, update the exception message in the single-segment case to match the new message:
https://github.com/dotnet/corefx/blob/aa6b47f9581eb7052b14a0a94db231938c12e599/src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs#L2186

Reply:
Done.

Reply:
Done at these two lines as well in the `SkipComment` method

https://github.com/dotnet/corefx/blob/87fdc75beacc76e02c8e572ae3b0613b418200eb/src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs#L2066

https://github.com/dotnet/corefx/blob/87fdc75beacc76e02c8e572ae3b0613b418200eb/src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs#L2072